crates/solidity/inputs/language/src/definition.rs

```diff
@@ -2603,26 +2603,19 @@ codegen_language_macros::compile!(Language(
                     items = [
                         Struct(
```

Comment:
From #652:

> Most of the lists dropped the List suffix specified in v0/v1

If it helps in migration, I'm happy to submit a PR to remove the `List` suffix in v0/v1.

Reply:
I've done a bit of work already bring v2 to be in line with v0/v1 where applicable (names, structure) in #650 for the parser codegen. This should also lessen the impact of v0->v2 spec migration, since we use the v0 names there already.

I know the names aren't perfect and not in line with what you'd expect from a Rust/typed AST API, but could we leave the renaming once we start using the new definition for the parser (and possibly the spec)? I'd rather we have a single definition as soon as possible and start evolving it from there, without introducing more accidental changes on the way :pray:

Reply:
I'm a bit worried about that in #650, since now it rewrites most of the parser rules. I was suggesting I can change v0/v1 separately to bring it up to date to the "desired" shape before the migration. However, I see now that the refactoring is already done, so no need to spend more time on it. I will try to go over it and collect other work items to be added to #655

Thanks for looking into it!